playbooks/loki.yaml

```
@@ -0,0 +1,44 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Configure loki.
+---
+- name: Configure loki
+  hosts: loki_servers
+  become: true
+  tasks:
+    - name: Install loki
+      ansible.builtin.package:
+        name: loki
+        state: present
+
+    - name: Configure loki
+      ansible.builtin.template:
+        src: files/loki/config.yml.j2
+        dest: /etc/loki/config.yml
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Install ferm configuration
+      ansible.builtin.template:
+        src: files/loki/ferm.conf.j2
+        dest: /etc/ferm/ferm.d/svc-loki-{{ item }}.conf
```

the `svc-` prefix is managed by the ferm role which will later remove this file because it looks like an outdated configuration for a service not enabled any more.
```suggestion
        dest: /etc/ferm/ferm.d/loki.conf
```